frontend/src/pages/Admin/__tests__/index.js

```diff
@@ -48,4 +48,28 @@ describe('Admin landing page', () => {
     const grantView = await screen.findByText('Select a user...');
     expect(grantView).toBeVisible();
   });
+
+  it('displays the flags page', async () => {
```

Comment:
These need to be added to increase coverage?

Reply:
yes indeed